content/docs/1.10.0/nodes-and-volumes/nodes/multidisk.md

```diff
@@ -5,47 +5,111 @@ weight: 4
 
-Longhorn will identify the disk using the path, so the users need to make sure the alternative path are correctly mounted when the node reboots, e.g. by adding it to `fstab`.
+Longhorn identifies disks by their paths, so you must ensure the alternative path is properly mounted after a node reboot, for example, by adding it to `fstab`.
 
 ## Remove a Disk
-Nodes and disks can be excluded from future scheduling. Notice that any scheduled storage space won't be released automatically if the scheduling was disabled for the node.
 
-In order to remove a disk, two conditions need to be met:
-- The scheduling for the disk must be disabled
-- There is no existing replica using the disk, including any replicas in an error state. About how to evict replicas from disabled disks, refer to [Select Disks or Nodes for Eviction](../disks-or-nodes-eviction/#select-disks-or-nodes-for-eviction)
+Nodes and disks can be excluded from future scheduling. Note that any storage already scheduled on the node will not be automatically released when scheduling is disabled for the node.
+
+To remove a disk, the following conditions must be met:
+
+- Scheduling for the disk must be disabled.
+- There must be no replicas or backing images remaining on the disk, including replicas in an error state. For instructions on how to evict replicas from disabled disks, see [Select Disks or Nodes for Eviction](../disks-or-nodes-eviction/#select-disks-or-nodes-for-eviction).
 
-Once those two conditions are met, you should be allowed to remove the disk.
+Once these conditions are satisfied, you can proceed to remove the disk.
 
 ## Configuration
+
 There are two global settings affect the scheduling of the volume.
 
-- `StorageOverProvisioningPercentage` defines the upper bound of `ScheduledStorage / (MaximumStorage - ReservedStorage)`. The default value is `100` (%). This translates to 150 GiB for Longhorn volumes that can be scheduled on a 200 GiB disk with 50 GiB reserved for the root file system.  A workload typically does not fill a volume completely, and Longhorn stores the volume contents as sparse files, so users might find that this setting can be increased.
-- `StorageMinimalAvailablePercentage` defines the lower bound of `AvailableStorage / MaximumStorage`.  The default value is `25` (%).  A volume cannot be scheduled on a disk if the volume's specified size would cause the available storage to fall below the threshold.
+- `StorageOverProvisioningPercentage` defines the maximum total storage that can be **scheduled** on a disk, relative to its usable capacity. The formula is:
+
+    ```bash
+    ScheduledStorage / (MaximumStorage - ReservedStorage)
+    ```
+
+    The default is `100` (%).
+
+    For a 200 GiB disk with 50 GiB reserved for the system, the usable capacity is 150 GiB. With the default setting, Longhorn can schedule up to 150 GiB of volume data on this disk.
```

Comment:
```suggestion
		On a 200 GiB disk with 50 GiB reserved, Longhorn sees 150 GiB of usable space. With the default setting, it can schedule up to 150 GiB of volume data.
```